src/core/components/Conversation/ImageCarousel.tsx

```diff
@@ -26,13 +32,13 @@ export const ImageCarousel = ({ imageIDs }: ImageCarouselProps) => {
     showSlides(slideIndex + n);
   };
 
-  // Function for specific slide control
+  //  Function for specific slide control
   const currentSlide = (n: number) => {
     showSlides(n);
   };
 
   return (
-    <div>
+    <div className={isLoaded ? styles.visible : styles.hidden}>
```

Comment:
The carousel is there the whole time, but once the first image is loaded and that state update is made, it will become visible.